src/ui/connection_profiles.ui

```diff
@@ -2192,37 +2219,46 @@
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
-           <property name="title">
-            <string>Required</string>
+           <property name="styleSheet">
+            <string notr="true"/>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <property name="topMargin">
              <number>9</number>
             </property>
-            <item row="0" column="0">
```

Comment:
:information_source: It is a PITA but the Qt Designer plug-in / standalone utility does not keep the order of `QGridLayout` members in any sort of order when it's contents are edited - which makes it hard to review changes therein in Git/Github!

You will likely have to manually reorder (into ascending row and then column order) using the "Open With" -> "Plain Text Editor" option and then cut and paste the `<item row="`X`" column="`Y`">` blocks by hand to restore the order and minimise the *noise* in the difference - fortunately you can fold the blocks and cut-and-paste those:
![screenshot from 2018-08-18 17-10-32](https://user-images.githubusercontent.com/6163092/44301174-4fe1d200-a30a-11e8-86ac-a16c06253e48.png)

Although: pasting as the **first** block will *mess up the indenting* (nasty :rage:) and pasting as the last block causes all the pasted lines to un-fold themselves (irritating :slightly_frowning_face:).  For safety's sake - make a copy of the file before you start hacking it manually in case it goes wrong!

Both this and the following profile preferences dialog/form will want this sort of clean-up I am afraid and it is advise-able because leaving the order wrong will make it much harder to resolve conflicts in other PRs that modify the same forms/dialogues (and the profile preferences one is a likely target for modifications) :astonished:

Reply:
I don't think manually reordering things around is something we should have everyone doing - especially since it's a lot easier to visualise changes by actually viewing the UI as it would look like instead of reading XML.

Reply:
Okay *visualising* the changes is good (before and after screen shots could be good here on GitHub) but the changes to the ordering makes it a nightmare to resolve conflicts when merging different PRs that both change this order in the same area of the file - and not keeping things in order generally makes the area of the changes *much* larger.

I agree that it is not something that should be done with loins that are *ungirded* - and that is why I urge making a backup before attempting it - so that you have something to go back to if Qt Designer cannot reopen the file after the hand tweaking (which an editor would of course do, {*without saving on success*} to check it still looks right afterwards)! :open_mouth: